boards/holybro/kakuteh7/init/rc.board_sensors

```diff
@@ -4,10 +4,15 @@
 #------------------------------------------------------------------------------
 board_adc start
 
-# The default IMU is an ICM20689, but there might also be an MPU6000
+# The KakuteH7 v1 comes by default with an ICM20689,
+# but there might also be an MPU6000
 if ! mpu6000 -R 6 -s start
 then
-	icm20689 -R 6 -s start
+	if ! icm20689 -R 6 -s start
+	then
+		# The Kakute H7 v2 comes with a BMI270 instead
+		bmi270 -s start
```

Comment:
I don't suppose there's any way to explicitly know if we're on a v1 or v2?

Reply:
@vincentpoont2 is there an easy way to check if we're on v1 or v2?

`ver all` says `HW arch: HOLYBRO_KAKUTEH7

Reply:
@julianoes Physically, they look different, there is also silkscreen on the PCB. 

Kakute ships with betaflight firmware, to use PX4, user need to flash PX4bootloader into it. 
If they share the same bootloader I guess there is no way to distinguish them.

Reply:
I have created separate build targets and `device_id`s now.